11ty/CustomLiquid.ts

```diff
@@ -243,7 +243,7 @@ export class CustomLiquid extends Liquid {
           $("figcaption").each((i, el) => {
             const $el = $(el);
             if (!$el.find("p").length) $el.wrapInner("<p></p>");
-            $el.prepend(`Figure ${i + 1}`);
+            $el.prepend(`<span>Figure ${i + 1}</span>`);
```

Comment:
If we want to try @iadawn's suggestion of inlining the "Figure N", it would go something like this, in order to insert it into the first paragraph within the figcaption:

```suggestion
            $el.find("p").first().prepend(`Figure ${i + 1} &mdash; `);
```

(To be clear though, RE @iadawn's comment, "Figure N" was already separated like this prior to this PR)